api/src/sockets/kots/DeploySocket.ts

```diff
@@ -135,12 +135,6 @@ export class KotsDeploySocketService {
 
     switch (app.restoreUndeployStatus) {
       case UndeployStatus.InProcess:
-        // retry undeploy every minute since socket.io is not bi-directional
-        const lastUndeployInterval = new Date().getTime() - this.lastUndeployTime;
```

Comment:
Removing this because running undeploy twice causes a race condition that causes the restored app to be deleted and the next restore to fail.